src/test/parameters.test.ts

```diff
@@ -21,7 +21,7 @@ describe("Parameters", () => {
   afterEach(() => nock.cleanAll());
 
   it("allow custom query params", async () => {
-    const productClient = new ShopperProducts({
+    const productClient = new ShopperProducts.ShopperProducts({
```

Comment:
All node SDK classes are wrapped in a namespace so the constructor is accessible from namespace.classname()